demo/index.html

```diff
@@ -16,45 +16,70 @@
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <link rel="import" href="../iron-ajax.html">
   <link rel="import" href="../../iron-image/iron-image.html">
-  <link rel="import" href="../../paper-styles/demo-pages.html">
-  <style>
-    iron-image {
-      background-color: lightgray;
-      margin: 1em;
-    }
-    .horizontal-section {
-      max-width: 300px;
-      margin-bottom: 24px;
-    }
-  </style>
 </head>
 <body unresolved>
   <h1>Video Feed</h1>
-  <div class="horizontal-section-container">
-    <template is="dom-bind" id="app">
+  <dom-module id="iron-ajax-demo">
+    <template>
+      <custom-style include="iron-flex">
+        <style>
```

Comment:
because I need to THROW IT AWAY... I actually should just remove the custom-style tag because i removed the dependency that provided iron-flex and just shoved the appropriate styles in there